website/mails/mails.py

```diff
@@ -107,7 +112,8 @@ def send_mail(to_addr, mail, mimetype='plain', from_addr=None, mailer=None,
         login=login,
         username=username,
         password=password,
-        mail_server=mail_server
+        mail_server=mail_server,
```

Comment:
It's weird to pass `mail_server` down through `send_mail` without parallel-ly passing down `sendgrid_api_key`. Seems like we should either stop passing the one, or start passing the other.